### PR TITLE
JS-DOM

### PR DESCRIPTION
**Pull request by NastjonkaK**
[Demo](https://nastjonkak.github.io/JS-DOM/) |
[Code base](https://github.com/NastjonkaK/JS-DOM)

The code is submitted in its own sub-directory and in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/NastjonkaK/JS-DOM/script.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    img: "linear-gradient(rgba(0,0,0, 0.7), rgba(0,0,0, 0.7)), url(Sourse/Croatia.jpg)",
+    description:
+      "Хорватія — це невелика південна країна з багатою культурною спадщиною, красивою природою і смачною кухнею. Це справжня перлина Адріатики з прекрасними пляжами Істрії та Далмації, теплим смарагдовим морем, чарівними стародавніми приморськими містами, оливковими гаями та вершинами Динарських гір."
+  }
+];
+
+
+const container = document.querySelector('.container');
+
+function changeContent (el){
```
Comment (A-Ostrovnyy) [inline submissions/NastjonkaK/JS-DOM/script.js]: Try to use Destructuring assignment [Link#1](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) [Link#2](https://learn.javascript.ru/destructuring).

File: submissions/NastjonkaK/JS-DOM/script.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+	let country = countries.find(obj => obj.id === el.target.id);
+	document.querySelector(".countryName").innerHTML = country.name;
```
Comment (A-Ostrovnyy) [inline submissions/NastjonkaK/JS-DOM/script.js]: Use Prettier for formatting code.

File: submissions/NastjonkaK/JS-DOM/script.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  document.querySelector(".shortDescription").innerHTML = country.description;
+  container.style.backgroundImage = country.img;
+};
+
+const chooseCountry = document.querySelector(".countries");
```
Comment (A-Ostrovnyy) [inline submissions/NastjonkaK/JS-DOM/script.js]: Variable declaration in global scope better write at the top of the document.

File: submissions/NastjonkaK/JS-DOM/index.html [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+			<div class="description">
+				<h3 class="countryName"></h3>
+				<p class="shortDescription"></p>
+			</div>
+		</div>
+	</div>
+	<script src="script.js"></script>
+</body>
+
+</html>
```
Comment (A-Ostrovnyy) [inline submissions/NastjonkaK/JS-DOM/index.html]: Add an empty line at the end of each file. [Reason](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline?noredirect=1&lq=1)

File: submissions/NastjonkaK/JS-DOM/Style/style.css [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+	.menu {
+		margin-top: 0;
+	}
+
+	.content {
+		padding-bottom: 5%;
+	}
+
+	.countries li, .shortDescription {
+		font-size: 50%;
```
Comment (A-Ostrovnyy) [inline submissions/NastjonkaK/JS-DOM/Style/style.css]: Try to use more appropriate units for `font-size`.

File: submissions/NastjonkaK/JS-DOM/Style/style.css [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+	width: 100%;
+	margin-top: 10px;
+	padding: 10%;
+	background-color: #1aaaaa;
+	border-radius: 5px;
+	text-align: center;
+}
+
+.countries li:hover,
+.countries li:active {
```
Comment (A-Ostrovnyy) [inline submissions/NastjonkaK/JS-DOM/Style/style.css]: Styles on `:active` state applies when a user make action to interactive elements. If styles on `:hover` and `:active` will be the same how do you tell the user about clicking on that element?)

**Comment by NastjonkaK** [comment]
@A-Ostrovnyy  Thanks for the comments. I fixed everything and I'm waiting for your review.

File: submissions/NastjonkaK/JS-DOM/script.js [hunk 7]
```diff
@@ -1,51 +1,56 @@
...
+    img:
+      "linear-gradient(rgba(0,0,0, 0.7), rgba(0,0,0, 0.7)), url(Sourse/Croatia.jpg)",
     description:
       "Хорватія — це невелика південна країна з багатою культурною спадщиною, красивою природою і смачною кухнею. Це справжня перлина Адріатики з прекрасними пляжами Істрії та Далмації, теплим смарагдовим морем, чарівними стародавніми приморськими містами, оливковими гаями та вершинами Динарських гір."
   }
 ];
 
+function changeContent(el) {
+  let { name, img, description } = countries.find(
+    obj => obj.id === el.target.id
```
Comment (Roka20012) [inline submissions/NastjonkaK/JS-DOM/script.js]: Hi) 👋 Nice job) 👍 
About destructing you can do this ```function changeContent({ target })``` it's looks better than ```el.target.id``` and in ```countries.find({ id }) ```

Comment (NastjonkaK) [reply submissions/NastjonkaK/JS-DOM/script.js]: Thanks for a tip! :-)

Comment (Roka20012) [reply submissions/NastjonkaK/JS-DOM/script.js]: You're welcome :)

**Comment by Roka20012** [comment]
@A-Ostrovnyy  hey hey☀️
Could you merge this pull request please?
Thanks🙏

**Comment by A-Ostrovnyy** [comment]
Well done!

**Pull request merged**